beta-component-promotions/2021.06-promotions/2021.06-promotions.md

```diff
@@ -0,0 +1,42 @@
+# PatternFly Beta Component Promotion Candidates
+## For Release 2021.08
+### [Banner](https://www.patternfly.org/v4/components/banner)
+A banner is a one-line, full color, full width container that can be used to communicate short snippets of information to users. Banners are un-intrusive, non-dismissible, and only support link buttons.
+
+![banner](./img/banner.png)
+
+### [Button: inoperableEvents](https://www.patternfly.org/v4/components/button)
+Button prop. An array of events to prevent when the button is in an aria-disabled state.
+
+### [Button: isAriaDisabled](https://www.patternfly.org/v4/components/button)
+Button prop. Adds disabled styling and communicates that the button is disabled using the aria-disabled html attribute
```

Comment:
```suggestion
Button prop. Adds disabled styling and communicates that the button is disabled using the aria-disabled HTML attribute.
``` 
Some lines have periods, and others don't. So I'm just adding periods for consistency 👍